.github/workflows/conformance-externalworkloads-v1.10.yml

```diff
@@ -65,7 +65,7 @@ env:
   vmName: ${{ github.repository_owner }}-${{ github.event.repository.name }}-${{ github.run_id }}-vm
   zone: us-west2-a
   vmStartupScript: .github/gcp-vm-startup.sh
-  cilium_cli_version: v0.10.3
+  cilium_cli_version: v0.10.4
```

Comment:
Don't we need to set the base version here as well?

Reply:
Good catch, I missed this one. Changed to set base version v1.10.